.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Can we put `.vscode` in a [`.npmignore`](https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html)?